ceph-extra-exporter/executer.go

```diff
@@ -95,29 +94,22 @@ func (ce *cephExecuter) update() {
 
 	jsonBytes, err := executeCommand(ce.rule.command, nil)
 	if err != nil {
-		_ = logger.Warn("command execution failed", map[string]interface{}{
-			"command": ce.rule.command,
-		})
+		logger.Warn("command execution failed", "command", ce.rule.command)
 		ce.failedCounter["command"] += 1
 		return
 	}
 
 	for name, metric := range ce.rule.metrics {
 		result, err := executeCommand([]string{"jq", "-r", metric.jqFilter}, bytes.NewBuffer(jsonBytes))
 		if err != nil {
-			_ = logger.Warn("jq command failed", map[string]interface{}{
-				"filter": metric.jqFilter,
-			})
+			logger.Warn("jq command failed", "filter", metric.jqFilter)
 			ce.failedCounter["jq"] += 1
 			continue
 		}
 
 		mv := []metricValue{}
 		if err := json.Unmarshal(result, &mv); err != nil {
-			_ = logger.Warn("parse value failed", map[string]interface{}{
-				"value": string(result),
-				"error": err,
-			})
+			logger.Warn("parse value failed", "value", result, "error", err)
```

Comment:
Could you tell me why you changed `string(result)` to `result`?

IMO, I prefer `string(result)` because `result` comes from the output of the `jq` command, so we can expect that it can be converted to the string type variable.

Reply:
I mistakenly thought that `result` was already a string. I'll fix it to use `string(result)`.